smac/scenario/scenario.py

```diff
@@ -250,12 +255,24 @@ def extract_instance_specific(instance_list):
                     self.feature_fn)[1]
 
         if self.feature_dict:
-            self.n_features = len(
-                self.feature_dict[list(self.feature_dict.keys())[0]])
             self.feature_array = []
             for inst_ in self.train_insts:
                 self.feature_array.append(self.feature_dict[inst_])
             self.feature_array = numpy.array(self.feature_array)
+            
+            # reduce dimensionality of features of larger than PCA_DIM
+            if self.feature_array.shape[1] > self.PCA_DIM:
+                X = self.feature_array
+                # scale features
```

Comment:
I haven't found the scaling in the PCA code; I found only centering around the mean which is not sufficient. Could you please point me to the exact code line for the scaling?
The MinMaxScaler of sklearn is also only doing the exact same stuff as I do in one line -- we save lines for imports, init of object and fit_transform call.